contracts/acl/ACL.sol

```diff
@@ -409,28 +471,31 @@ contract ACL is IACL, TimeHelpers, AragonApp, ACLHelpers {
         return r2; // both or and and depend on result of r2 after checks
     }
 
-    function compare(uint256 _a, Op _op, uint256 _b) internal pure returns (bool) {
-        if (_op == Op.EQ)  return _a == _b;                              // solium-disable-line lbrace
-        if (_op == Op.NEQ) return _a != _b;                              // solium-disable-line lbrace
-        if (_op == Op.GT)  return _a > _b;                               // solium-disable-line lbrace
-        if (_op == Op.LT)  return _a < _b;                               // solium-disable-line lbrace
-        if (_op == Op.GTE) return _a >= _b;                              // solium-disable-line lbrace
-        if (_op == Op.LTE) return _a <= _b;                              // solium-disable-line lbrace
-        return false;
-    }
-
-    function checkOracle(IACLOracle _oracleAddr, address _who, address _where, bytes32 _what, uint256[] _how) internal view returns (bool) {
+    /**
+    * @dev Internal function to perform an ACL oracle check
+    */
+    function _checkOracle(IACLOracle _oracleAddr, address _user, address _who, address _where, bytes32 _what, uint256[] _how)
+        internal
+        view
+        returns (bool)
+    {
         bytes4 sig = _oracleAddr.canPerform.selector;
 
         // a raw call is required so we can return false if the call reverts, rather than reverting
-        bytes memory checkCalldata = abi.encodeWithSelector(sig, _who, _where, _what, _how);
+        bytes memory checkCalldata = abi.encodeWithSelector(sig, _user, _who, _where, _what, _how);
+        return _checkOracle(_oracleAddr, checkCalldata);
+    }
 
+    /**
+    * @dev Internal function to perform an ACL oracle check
+    */
+    function _checkOracle(IACLOracle _oracleAddr, bytes memory _callData) internal view returns (bool) {
```

Comment:
Could we inline this function with the one before? I don't think it's worth having two internal ones that just use each other :).